app/controllers/allocations_controller.rb

```diff
@@ -1,30 +1,23 @@
 class AllocationsController < ApplicationController
   before_action :authenticate_user
 
+  alias_action :new, :edit
+
   def new
-    @prisoner = OffenderService.new.get_offender(nomis_offender_id_from_url)
+    @prisoner = get_prisoner(nomis_offender_id_from_url)
     @recommended_pom = @prisoner.current_responsibility
```

Comment:
Is this actually recommended_pom_type?

Reply:
Yeah I think this should be renamed - will change